symfony/plugins/orangehrmAttendancePlugin/Controller/ViewEmployeeAttendanceController.php

```diff
@@ -60,9 +60,18 @@ public function preRender(Request $request): void
             //If the logged-in user is an Admin, regardless of the configuration,
             // he is able to add/edit/delete employee attendance records
             if (
-                ($this->getAuthUser()->getUserRoleName() === self::USER_ROLE_ADMIN ||
-                    $this->getAttendanceService()->canSupervisorModifyAttendanceConfiguration()) &&
-                $this->getAuthUser()->getEmpNumber() !== $empNumber
+                (
+                    $this->getAuthUser()->getUserRoleName() === self::USER_ROLE_ADMIN &&
```

Comment:
better to use workflow state machine, rather than hard coding user role

Reply:
done